.circleci/config.yml

```diff
@@ -179,19 +199,22 @@ workflows:
     test_all:
         jobs:
           - test_optimus
-          - test_optimus_mouse
-          - test_smartseq2
-          - test_smartseq2_single_end
-          - test_emptyDropsWrapper
-          - test_npz2rds
-          - test_snap_atac
-          - test_sc_atac
-          - test_bulk_rna
-          - test_portability_snap_atac
-          - test_portability_bulk_rna_pipeline
-          - test_portability_optimus
-          - test_portability_optimus_mouse
-          - test_portability_smartseq2_single_sample
-          - test_optimus_v3
-          - test_smartseq2_multisample
-          - test_smartseq2_multisample_single_end
+          - test_optimus_snrna
```

Comment:
should '-test_optimus_snrna_v3' also be included?

Reply:
We are not changing any v3 specific code so that path should not be affected at all. All tests should be reenabled before merging